static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -44,6 +47,7 @@ const isViewable = (v: Variant, t: ABTest): boolean => {
  * acquisition tests in priority order (highest to lowest)
  */
 export const acquisitionsTests: $ReadOnlyArray<AcquisitionsABTest> = [
+    acquisitionsEpicOptimizeAATest, // FIXME
```

Comment:
is it meant to only be added if the switch is on?

Reply:
FIXME can probably be removed. The process of selecting a test is to iterate through the array until a viable test is found. But I'm not sure if we want to give this test priority over the Google doc tests. I'll check with Jesse.

Reply:
Spoke to Jesse. We want to give this test top priority i.e. it can stay where it is. (So just need to remove the FIXME @joelochlann (GDRP))